### DISCUSSION TITLE
signup/login manager

### DISCUSSION THREAD

**Discussion by AbhimanyuAryan** [GenieFramework/Genie.jl#499] [Ideas]
This is a feature request/offer: I wrote Genie code to use ORCID (https://orcid.org/) as a signup/login manager, is this something that would be of interest as a Genie plugin?  [ From: Genie Framework Discord by @tpoisot ]

**Comment by tpoisot** [comment]
I'm still working on making it work with @authenticated! and all that, but I'll update when I have something to show

**Comment by essenciary** [comment]
@tpoisot let us know if we can help! Thanks for your effort!
